.github/workflows/publish_binaries.yaml

```diff
@@ -0,0 +1,25 @@
+name: publish-binaries
```

Comment:
Can this be a part of the existing tag workflow? Would be nice to have the Github release created via tag along with the container pushes.

Reply:
I think GoReleaser creates the Github release automatically. It does on the conftest repo without any other GH actions.

https://github.com/open-policy-agent/conftest/blob/master/.github/workflows/release.yaml#L49

Reply:
Ah good point. I made the changes.